src/migrations/20230510113903-fix-api-token-username-migration.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+        SELECT * FROM migrations WHERE name = '/20230426110026-rename-api-token-username-field';
+        `,
+        (err, results) => {
+            if (results.rows.length > 0) {
+                db.runSql(
+                    `
+                ALTER TABLE api_tokens RENAME COLUMN token_name TO username;
+                ALTER TABLE api_tokens ADD COLUMN "token_name" text;
+                UPDATE api_tokens SET token_name = username;
```

Comment:
Wouldn't this be the same but without the RENAME?
```suggestion
                ALTER TABLE api_tokens ADD COLUMN "username" text;
                UPDATE api_tokens SET username = token_name;
```

Reply:
The reason for the rename is that the original `username` column is NOT NULL - and the added column becomes nullable, so we keep the contract to keep old clients running. They would be getting non-null errors on `token_name`

Reply:
Ok now I get it! If you run `ALTER TABLE api_tokens ADD COLUMN "username" text NOT NULL;` it will fail because there are rows in the table